### PR TITLE
Update to new versions of jakarta APIs and default implementations

### PR DESCRIPTION
**Pull request by jhanders34**
- Updates to Jakarta API bundles that have a new micro version
- Updates to mail, jaxws tools, glassfish json, soap and activation implementation that have a new micro version

#build

### CODE REVIEW

**Review by tkburroughs — DISMISSED** [review]

**Review by neuwerk — DISMISSED** [review]
LGTM, thank you!

**Review by neuwerk — DISMISSED** [review]
LGTM

File: dev/com.ibm.ws.org.glassfish.json.2.0/bnd.bnd [hunk 1]
```diff
@@ -10,11 +10,11 @@
...
--include= jar:${fileuri;${repo;org.glassfish:jakarta.json;2.0.0;EXACT}}!/META-INF/MANIFEST.MF,bnd.overrides
+-include= jar:${fileuri;${repo;org.glassfish:jakarta.json;2.0.1;EXACT}}!/META-INF/MANIFEST.MF,bnd.overrides
 
 -includeresource: \
-   @${repo;org.glassfish:jakarta.json;2.0.0;EXACT}!/org/glassfish/json/*
+   @${repo;org.glassfish:jakarta.json;2.0.1;EXACT}!/org/glassfish/json/*
 
 -buildpath: \
-	org.glassfish:jakarta.json;version=2.0.0
+	org.glassfish:jakarta.json;version=2.0.1
```
Comment (tevans78) [inline dev/com.ibm.ws.org.glassfish.json.2.0/bnd.bnd]:
```suggestion
	org.glassfish:jakarta.json;version=2.0.1;strategy=exact
```

**Review by jonhawkes — APPROVED** [review]

**Review by tevans78 — APPROVED** [review]

**Review by njr-11 — APPROVED** [review]

**Comment by jhanders34** [comment]
#libby #build

**Pull request merged**